backend/backend/api/routes.py

```diff
@@ -130,12 +130,12 @@ async def read_root():
     </style>
 </head>
 <body>
-    <h1>Test Result</h1>
-    <input type="text" id="testInputText" placeholder="Enter your text here">
-    <button onclick="getTestOutput()">Test</button>
+    <h1>Research assistant</h1>
```

Comment:
Consider renaming the buttons from 'Test' and 'Tell Joke' to more descriptive names, such as 'Run Research' and 'Generate Joke', respectively. This will improve user experience by clearly indicating what each button does.